yuniql-cli/JsonPrinter.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Yuniql.Core;
+
+namespace Yuniql.CLI
+{
+    /// <summary>
+    /// This class is used to print configuration variable as json text
+    /// </summary>
+    public class JsonPrinter : IPrinter
+    {
+        private readonly string[] titles =  new string[] { "property","value","source"};
+        private readonly List<string[]> rows = new List<string[]>();    
+        
+        public void AddRow(params object[] row)
+        {
+            if (row.Length != titles.Length)
+            {
+                throw new Exception($"Added row length [{row.Length}] is not equal to title row length [{titles.Length}]");
+            }
+            rows.Add(row.Select(o => o.ToString()).ToArray());
+        }
+
+        public void Print()
```

Comment:
Try to look at class `ConfigurationService`, there we have a method `PrintAsJson` using native JsonSerializer. I think its sufficient to display the results as json.

https://github.com/rdagumampan/yuniql/blob/b2cfbf5dc0068c2411bc034a95d12ddebab3bd5a/yuniql-core/ConfigurationService.cs#L154

Reply:
I guess its ok, because you are building here each row to print as json.